modules/rtpengine/rtpengine.c

```diff
@@ -1500,7 +1500,7 @@ static int connect_rtpengines(void)
 static int
 child_init(int rank)
 {
-	mypid = getpid();
+	myprefix+=getpid()+rand()%10000;
```

Comment:
Giving a deeper though about it, this approach might increase collision between instances that use different PIDs, or even between the same OpenSIPS instance, it increases the chances when `PID1+rand1 = PID2+rand2`. 
So probably the best idea is a combined one - have the cookie as `PID_RAND_MYSEQN`. If you don't have time to work this out by tomorrow (when we have a freeze), I will merge your PR and make the change myself, but if you do, we would all be grateful for your work :). Thanks!

Reply:
That's a good point, it could lead to the same situation in some cases. I will push your proposed schema shortly.